### ISSUE TITLE
flutter_test from sdk is incompatible with get_it >=7.1.4.

### ISSUE THREAD

**Issue by saigkl** [fluttercommunity/get_it#204]
```
[app] flutter pub get
Running "flutter pub get" in app...                         
Because every version of flutter_test from sdk depends on async 2.6.1 and get_it >=7.1.4 depends on async ^2.7.0, flutter_test from sdk is incompatible with get_it >=7.1.4.

So, because app depends on both get_it ^7.1.4 and flutter_test any from sdk, version solving failed.
pub get failed (1; So, because app depends on both get_it ^7.1.4 and flutter_test any from sdk, version solving failed.)
exit code 1
```

**Comment by henry-hz** [comment]
but it works with flutter-2.3-candidate.26

**Comment by escamoteur** [comment]
it also works when you are on mastr

**Comment by mauriceraguseinit** [comment]
Because every version of flutter_test from sdk depends on async 2.6.1 and get_it 7.1.4 depends on async ^2.7.0, flutter_test from sdk is incompatible with get_it 7.1.4.
So, because verbraucherschutz depends on both get_it 7.1.4 and flutter_test any from sdk, version solving failed.
pub get failed (1; So, because verbraucherschutz depends on both get_it 7.1.4 and flutter_test any from sdk, version solving failed.)

Flutter 2.2.2 with get_it 7.1.4 ... 

:-(

**Comment by escamoteur** [comment]
@mauriceraguseinit the latest change won't have an impact on you probably because it was a very rare edge case.

**Comment by mauriceraguseinit** [comment]
hmm.. ok now I use get_it 7.1.3

**Comment by sachin-dahal** [comment]
I was also facing the same issue with **flutter_test from sdk is incompatible** which is because of the get_it version ^7.1.4
I downgrade the version to ^7.1.3 and there is no issue
Hope, this issue would get fixed soon in the upcoming version of get_it

**Comment by escamoteur** [comment]
fixed with V7.2.0

**Comment by UsamaEjaz0** [comment]
```
Because every version of flutter_test from sdk depends on async 2.5.0 and get_it >=7.2.0 depends on async ^2.6.0, flutter_test from sdk is incompatible with get_it >=7.2.0.
So, because watermark depends on both get_it ^7.2.0 and flutter_test any from sdk, version solving failed.
pub get failed (1; So, because watermark depends on both get_it ^7.2.0 and flutter_test any from sdk, version solving failed.)
```
V7.2.0 still has this issue, v7.1.3 works fine.
